File: common-core/src/main/java/com/ityu/bean/entity/system/Relation.java
package com.ityu.bean.entity.system;

import lombok.Data;
import org.hibernate.annotations.Table;

import javax.persistence.*;

/**
 * Created  on 2018/4/2 0002.
 *
 * @author enilu
 */
@Entity(name = "t_sys_relation")
@Table(appliesTo = "t_sys_relation", comment = "菜单角色关系")
@Data
public class Relation {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column
    private Long menuid;
    @Column
    private Long roleid;

}

File: common-core/src/main/java/com/ityu/common/bean/entity/system/Relation.kt
package com.ityu.common.bean.entity.system

import jakarta.persistence.Table
import jakarta.persistence.*

/**
 * Created  on 2018/4/2 0002.
 *
 * @author enilu
 */
@Entity(name = "t_sys_relation")
@Table(name = "t_sys_relation", )
open class Relation {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
     open var id: Long? = null

    @Column
     open var menuid: Long? = null

    @Column
     open var roleid: Long? = null
}
